pyaedt/modeler/parts.py

```diff
@@ -360,9 +366,9 @@ def do_rotate(self, app, aedt_object):
         Parameters
         ----------
         app: pyaedt.Hfss
-            HFSS instance of AEDT.
-        aedt_object : str
-            Name of the design in AEDT.
```

Comment:
@akaszynski I just want to confirm so I'm not mistaken, but numpy docstrings spacing should be:

```
variable_name : str
    Description.
```
With the spaces in between the colon, correct?

Reply:
@akaszynski @jleonatti  This is a SIGNIFICANT question. The rendering looks very different between the two scenarios. With a space between the colon, the type is in a bigger, bolder blue font versus the gold font. I was removing all the spaces in the new content to get the newly added content to looking like the older content. Please advise what the formatting should be!

Reply:
Please keep the space between the variable name and the string.  There's an additional numpydoc validation that I'll turn on to make sure this is enforced.

Reply:
So for confirmation, you want "is_infinite_grd : bool, optional" rather than "is_infiniite_grd: bool, optional", correct? Is there a way to do a global search and replace of all documents in the repo? I ask because there will be a ton of changes needed!

Reply:
`is_infinite_grd : bool, optional` is correct.

Took 15 minutes with emacs and regex.  See: #470